pages/faqs/yes-you-can-faq.mdx

```diff
@@ -0,0 +1,196 @@
+---
+layout: 'guide'
+title: 'What can I do with Avo?'
```

Comment:
I'm wondering if we should add sub sections to this page to categorize the questions? Could be the same as in the header: Implementation, Data Design, Exploring the Tracking Plan